canonical-web-and-design/maas-ui
# feat(ui): add removing of interfaces

## Done

- Add the remove interface control to the network table

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the network tab for a machine with editable interfaces.
- Using the row menu click "Remove X" where X is the interface type.
- Click confirm.
- The interface should get removed.

## Fixes

Fixes: #2149.
